espresso-server/app/build.gradle.kts

```diff
@@ -0,0 +1,132 @@
+plugins {
+    id("com.android.application")
+    kotlin("android")
+}
+
+android {
+    compileSdkVersion(getIntProperty("appiumCompileSdk", 28))
+    buildToolsVersion(getStringProperty("appiumBuildTools", "28.0.3"))
+    defaultConfig {
+        // <instrumentation android:targetPackage=""/>
+        applicationId = getStringProperty("appiumTargetPackage", "io.appium.espressoserver")
+        // <manifest package=""/>
+        testApplicationId = "io.appium.espressoserver.test"
+        testHandleProfiling = false
+        testFunctionalTest = false
+        minSdkVersion(getIntProperty("appiumMinSdk", 18))
+        targetSdkVersion(getIntProperty("appiumTargetSdk", 28))
+        versionCode = 1
+        versionName = "1.0"
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        getByName("debug") {
+            zipAlignEnabled(getBooleanProperty("appiumZipAlign", true))
+        }
+        getByName("release") {
+            isMinifyEnabled = false
+            proguardFiles(getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro")
+        }
+    }
+
+    sourceSets {
+        getByName("test") {
+            java.srcDirs("src/androidTest/java")
+        }
+    }
+
+    testOptions {
+        unitTests.isReturnDefaultValues = true
+    }
+
+    signingConfigs {
+        getByName("debug") {
+            findProperty("appiumKeystoreFile")?.also {
+                storeFile(File(it.toString()))
+            }
+
+            findProperty("appiumKeystorePassword")?.also {
+                storePassword(it.toString())
+            }
+
+            findProperty("appiumKeyAlias")?.also {
+                keyAlias(it.toString())
+            }
+
+            findProperty("appiumKeyPassword")?.also {
+                keyPassword(it.toString())
+            }
+        }
+    }
+}
+
+dependencies {
+    implementation(fileTree(mapOf("dir" to "libs", "include" to listOf("*.jar"))))
+    // additionalAppDependencies placeholder (don't change or delete this line)
```

Comment:
Espresso driver dynamically modifies some placeholders like here by https://github.com/appium/appium-espresso-driver#espresso-build-config , so they also should be updated to this syntax. (Probably add `()`?)

Reply:
I fixed the additional dependencies and also I updated the documentation links.